web/src/SidebarTriggerButton.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { mount } from "enzyme"
+import SidebarTriggerButton from "./SidebarTriggerButton"
+import { TriggerMode } from "./types"
+
+it("renders trigger button in manual mode", () => {
+  const root = mount(
+    <SidebarTriggerButton
+      isSelected={true}
+      resourceName="doggos"
+      triggerMode={TriggerMode.TriggerModeManual}
+    />
+  )
+
+  expect(root.find(".SidebarTriggerButton")).toHaveLength(1)
```

Comment:
I think this should be a snapshot test. Otherwise it's not really testing that much.

Reply:
We could, though, test that it makes certain kinds of requests when clicked.